src/main/java/seedu/address/model/Model.java

```diff
@@ -157,4 +158,8 @@
      * @throws NullPointerException if {@code predicate} is null.
      */
     void updateFilteredCheeseList(Predicate<Cheese> predicate);
+
+    void setScreenNumber(int screenNumber);
```

Comment:
Would it be possible to use enums instead?

Reply:
Noted, will address this before tmr (monday) morning. Will probably be adding a class to Model to reflect the screens to show.